test/NuGet.Core.FuncTests/NuGet.Packaging.CrossVerify.Verify.Test/VerifySignedPackages.cs

```diff
@@ -346,6 +347,36 @@ public async Task VerifySignaturesAsync_PreGenerateSignedPackages_AuthorSigned_R
             }
         }
 
+        [Theory]
+        [MemberData(nameof(FolderForWindows_NetFulFramework))]
+        public async Task VerifySignaturesAsync_PreGenerateSignedPackages_AuthorSigned_TimeStampedWithNoSigningCertificateUsage_Throws(string dir)
+        {
+            // Arrange
+            var caseName = "ATNOCERTIFICATEUSAGE";
+
+            var signedPackageFolder = Path.Combine(dir, caseName, "package");
+            var signedPackagePath = Directory.GetFiles(signedPackageFolder).Where(f => f.EndsWith(".nupkg")).First();
+
+            using (var packageReader = new PackageArchiveReader(signedPackagePath))
+            using (var store = new X509Store(StoreName.Root,
+                RuntimeEnvironmentHelper.IsWindows ? StoreLocation.LocalMachine : StoreLocation.CurrentUser))
```

Comment:
Is there a good reason not to use [`Test.Utility.Signing.CertificateStoreUtilities.GetTrustedCertificateStoreLocation()`](https://github.com/NuGet/NuGet.Client/blob/32517d8e81b0bc1d143cb5080020e0b1769bb1c2/test/TestUtilities/Test.Utility/Signing/CertificateStoreUtilities.cs#L11)?